### PR TITLE
Space-Travelers-Hub

### PR DESCRIPTION
**Pull request by FrankMugagga**
### Activities
- Created Navigation bar to access the different pages
- Fetched data from the Rocket, Dragon and Mission APIs
- Dispatch actions for reserving rockets and dragons and joining missions
- Dispatch actions for cancelling rockets and dragons and leaving missions
- Created badges for reserving rockets and dragons as well as for joining missions
- Display joined missions, reserved rockets and dragons on a profile page
- Write test cases for missions, rockets and dragons
- Deploy the app for live preview
- Created a comprehensive read me file for the project details
- Setup and used Kanban board for this project

### CODE REVIEW

File: src/components/features/dragons.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  const handleReserveBtn = (id) => {
+    dispatch(reserveD(id));
+  };
+  const handleCancelBtn = (id) => {
+    dispatch(cancelD(id));
+  };
+
+  if (loading) {
+    return <div>loading....</div>;
+  }
```
Comment (aamirkhan2478) [inline src/components/features/dragons.js:25]: - [ ] Well done :clap:   you guys did a very nice job :+1: but there are some problems with your dragon page when we click on the `reserve button` it changes to the `cancel button` it's good :+1: but when we go to the next page and come back to the dragon page the state is empty so kindly fix this issue :+1: check below :point_down: my explanation video then you will understand better :slightly_smiling_face: 

[screen-capture (1).webm](https://user-images.githubusercontent.com/45226848/236560980-56360f02-9a39-47ed-93f7-89ea4de1189b.webm)

File: src/components/features/dragons.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      <div className="container_dragons">
+        {dragon.reserved && <p className="reserved">Reserved</p>}
+        <p>{dragon.description}</p>
+      </div>
+      {/* <span>
+<p className="reserved">{dragon.reserved && <p>'Reserved'</p>}</p>
+{' '}
+{dragon.description}
+</span> */}
```
Comment (aamirkhan2478) [inline src/components/features/dragons.js:50]: - [ ] Please remove unwanted commented code from your codebase it's your codebase make messy so kindly remove it and if you want your previous change the check the GitHub commit history :+1:

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @FrankMugagga , @kamranahmad123 & @sumon766 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters passed
✔️ All unit tests passed

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @FrankMugagga, @kamranahmad123 and @sumon766 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Use React components ✔️ 
Use React props ✔️ 
Use React Router ✔️ 
Connect React and Redux ✔️
Handle events in a React app ✔️
Use styles in a React app ✔️
Use React hooks ✔️
Use store, actions and reducers in React ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**